pyproject.toml

```diff
@@ -222,6 +222,10 @@ extend-ignore = [
     # pep8-naming (N)
     # NOTE: some of these can/should be fixed, but this changes the API.
     "N8",
+    "N999",  # Invalid module name                     # TODO: fix
```

Comment:
If we update ruff to `0.0.249` this will be fixed.

Reply:
I think it's worth bumping here.

Reply:
Done